x/tax/keeper/taxdecorator_test.go

```diff
@@ -67,6 +76,14 @@ func (suite *KeeperTestSuite) TestTaxDecorator() {
 			expPass:   false,
 			expErr:    sdkerrors.ErrInsufficientFunds,
 		},
+		{
+			title:     "pay fees with insufficient funds (not base denom) should fail",
```

Comment:
Rename the following test:
"pay fees with multiple denoms should fail"
to
"pay fees with multiple now allowed denoms should fail", or something similar.

Consider adding one more test:
"pay fees with multiple denoms where both base and allowed denoms are sufficient should [fail/succeed]"
"pay fees with multiple denoms where base denom is insufficient and allowed denom is sufficient should [fail/succeed]"
"pay fees with multiple denoms where both base and allowed denoms are insufficient should [fail/succeed]"